CHANGELOG.md

```diff
@@ -553,7 +556,13 @@ redundant code which was similar for APU2/3/5 boards.
 - turn off D4 and D5 leds on boot
 - enable power on after power failure
 
-[Unreleased]: https://github.com/pcengines/coreboot/compare/v4.14.0.4...develop
+[Unreleased]: https://github.com/pcengines/coreboot/compare/v4.16.0.1...develop
```

Comment:
Something is wrong here. Why do we add so much links?

Reply:
The links were missing in previous releases. For example, on the master branch, v4.14.0.4 has the proper link, but v4.14.0.5 has no link and is surrounded by square brackets. https://github.com/pcengines/coreboot/blob/release/CHANGELOG.md
This change should fix the missing links.